File: screens/book.js
import React from 'react';
import { View, Text, Image, Button } from 'react-native';
import { globalStyles } from '../styles/global';
import { useNavigation } from '@react-navigation/native';

export default function Book() {
    const { navigate } = useNavigation();
    return (
        <View style={globalStyles.container}>
            <Text>Book Screen</Text>
            <Image
                style={globalStyles.thumbnail}
                source={require("../assets/unnamed.jpg")} />
            <Text style={globalStyles.paragraph}>
                Title:
            </Text>
            <Text>
                Author(s):
            </Text>
            <Text>
                Published:
            </Text>
            <Text>
                Description:
            </Text>
            <Text>
                Page Count:
            </Text>
            <Text>
                ISBN: (ISBN_13,ISBN_10,OTHER)
            </Text>
            <Button title='Click here for more information' /*onPress={bookInformationLink}*/ />
            <Text>
                Index : name.id,
            </Text>
            <Text>
                Categories: name.volumeInfo.categories
            </Text>
            <View>
                <Button title='Wishlist' onPress={() => navigate('WishList')}/>
                <Button title='Add To Library' />
            </View>
        </View>
    );
}

File: ReadingRainbowAPI/Models/Book.cs
using System.Collections.Generic;

namespace ReadingRainbowAPI.Models
{
    public class Book : Neo4jEntity
    {
        public string Id {get; set;}
        public string BookInformationLink {get; set;}
        public string Title {get; set;}
        public List<string> Authors = new List<string>();
        public string Thumbnail {get; set;}
        public string SmallThumbnail {get; set;}
        public string PublishDate {get; set;}
        public string NumberPages {get; set;}
        public string Description {get; set;}
        public string ISBN_10 {get; set;}      
        public string ISBN_13 {get; set;}   
        public List<string> Cateogries = new List<string>();

    }   
}